packit/utils/decorators.py

```diff
@@ -1,13 +1,14 @@
 import functools
 import logging
-from typing import Union, Tuple
+from typing import Union, Tuple, Type, Any
 
 logger = logging.getLogger(__name__)
 
 
 def fallback_return_value(
-    fallback_value, exceptions: Union[Exception, Tuple[Exception]] = Exception()
-):
+    fallback_value: Any,
+    exceptions: Union[Type[Exception], Tuple[Type[Exception]]] = Exception,
+) -> Any:
```

Comment:
can we have a test case for this? :O

Reply:
I can try to add a test case for the decorator but not exactly for this "bug". I think mypy should have been shouting about this so I may try to create a minimal example and file an issue there.

Reply:
Never mind, mypy is correct, but our closure below did not have types, hence mypy did not lint it and detect error :D Maybe we should enforce typing using `mypy --disallow-untyped-defs` or similar (though that may require some extra work)

Reply:
No test to be added (the decorator is already tested). I've added typing to the inner functions so now mypy would throw an error without the fix of `exceptions` argument type.